src/blocks/gallery-carousel/edit.js

```diff
@@ -229,6 +218,21 @@ class GalleryCarouselEdit extends Component {
 			);
 		}
 
+		const flickityOptions = {
+			draggable: false,
+			pageDots: true,
+			prevNextButtons: true,
+			wrapAround: true,
+			autoPlay: false,
+			cellAlign: alignCells ? 'left' : 'center',
```

Comment:
This doesn't seem to be re-positioning the cells to the left or center when the "Align Cells" slider setting is toggled in the editor. However, if you save the post and reload the editor it works. No problems on the theme side.

Reply:
Yea I think it’s acceptable as is for now.